### ISSUE TITLE
Discrepancy in TypeScript types

### ISSUE THREAD

**Issue by tearwyx** [pinojs/pino-pretty#273]
According to the module `pino-abstract-transport` the function exported by default returns a value of type `Transform` from the module `stream`, but in the declaration file it returns type `(inputData: string | object) => string`.

Based on version 7.2.0.

**Comment by mcollina** [comment]
Thanks for reporting!
Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by tearwyx** [comment]
@mcollina #274

**Comment by SushiWaUmai** [comment]
Having the same issue with the type Prettifier
```
declare namespace PinoPretty {
  type Prettifier = (inputData: string | object) => string;
  type MessageFormatFunc = (log: LogDescriptor, messageKey: string, levelLabel: string) => string;
  type PrettyOptions = PrettyOptions_
}
```

**Comment by kibertoad** [comment]
@SushiWaUmai Can you send a PR with tests?

**Comment by tearwyx** [comment]
If it is the same issue, there is no need to fix it once it has been fixed in the main branch (#274) and hopefully it will get out someday.
